config/swagger.yaml

```diff
@@ -402,6 +402,434 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security:
+  /accounts/{pubkey}:
+    get:
+      tags:
+        - external
+        - gossip
```

Comment:
Not sure why gossip tag?

Reply:
Leftovers from previous tagging. We should re-evaluate all tags usage while cleaning up unused endpoints.

Reply:
Also tracked here: https://www.pivotaltracker.com/n/projects/2124891/stories/158872647